README.md

```diff
@@ -199,16 +199,16 @@ Container images for amazon-eks-pod-identity-webhook can be found on [Docker Hub
 
 ### In-cluster
 
-You can use the provided configuration files in the `deploy` directory, along with the provided `Makefile`
+You can use the provided configuration files in the `deploy` directory, along with the provided `Makefile`. You must install a cert-manager as it is a pre-requisite for below deployments. (See [cert-manager installation](https://cert-manager.io/docs/installation/)) 
```

Comment:
I think it should be in a `prerequisite` section to make it stand out.

Reply:
added it in prerequisites sub-section under `Installation`